newIDE/app/src/ObjectsRendering/Renderers/Rendered3DInstance.js

```diff
@@ -92,6 +94,7 @@ export default class Rendered3DInstance {
     if (this._pixiObject !== null)
       this._pixiContainer.removeChild(this._pixiObject);
     if (this._threeObject !== null) this._threeGroup.remove(this._threeObject);
+    this._wasDestroyed = true;
```

Comment:
should this be called at the top of the function? It should synchronous, but in case this function is called, wondering if we should asap prevent anything else

Reply:
JS being mono threaded, I think it's ok but I will move it at top to ensure it's safe.

Reply:
(in the sense that whenever this function onRemovedFromScene is launched, NOTHING will stop it from being run until the end, even if there are promises ready and waited to be completed... but this could be helpful if one day this function does some asynchronous work)